entities/A_Station.php

```diff
@@ -35,6 +36,10 @@ class Station {
    */
   public static function fetchAll() {
     $ss = DB::instance()->stations()->fetchAll();
+    if(is_bool($ss))
```

Comment:
This is technically correct, but I think a "more correct" way to do it might be 
```php
if(!$ss){
  //Throw error
}
```

If you look around at other PHP code I suspect this is what you most often see. (I suspect php returns a boolean specifically so you CAN use the !$result syntax.)